src/content/docs/guides/Audio/0-getting-started-with-audio.mdx

```diff
@@ -289,6 +291,115 @@ do
 CloseAllWindows();
 ```
 
+</TabItem>
+<TabItem label="Object-Oriented">
+
+```csharp
+using SplashKitSDK;
```

Comment:
Please do 
```
using SplashKitSDK;
using static SplashKitSDK.SplashKit;
```

For the C# like how it is in the top level. You can fix this quickly by removing all occurrences of `SplashKit.`